docs/user-guide/ui/access/vpn.md

```diff
@@ -8,16 +8,16 @@ If you don't know what VPN is; below you can read a little description you can f
 
 _A Virtual Private Network, or VPN, is a special SSL encrypted connection over the Internet that connects a device to a different network, such as your home or work local network (LAN) when you are not physically connected to this private network. The encrypted connection helps ensure that sensitive data is safely transmitted. It prevents unauthorized people from eavesdropping on the traffic and allows the user to conduct work remotely as if they were sitting right next to their node even if it's actually on the other side of the world in a datacenter that's inaccessible to the public at all times.
 There are 2 major use cases for a VPN. For our use of VPN(s) with DAppNode we use the VPN to be able to securely connect to the private internal Docker network of your DAppNode, a private network that exists within your DAppNode, which is how you can access and navigate the Admin UI and all other package specific UI's that are unavailable using the Local Proxy by creating a direct virtual encrypted tunnel that makes your client device part of the internal DAppNode private network.  
-The other primary use of a VPN which yu may already know is a privacy use case where users purchase subscriptions from VPN providors who instead of connecting your tunnel to a specific private LAN they create a tunnel from your client device to their VPN server which then allows you to browse the Internet from a different goegraphic location and IP address than you would without a VPN_
+The other primary use of a VPN which you may already know is a privacy use case where users purchase subscriptions from VPN providors who instead of connecting your tunnel to a specific private LAN they create a tunnel from your client device to their VPN server which then allows you to browse the Internet from a different goegraphic location and IP address than you would without a VPN._
```

Comment:
```suggestion
The other primary use of a VPN which you may already know is a privacy use case where users purchase subscriptions from VPN providers who instead of connecting your tunnel to a specific private LAN they create a tunnel from your client device to their VPN server which then allows you to browse the Internet from a different goegraphic location and IP address than you would without a VPN._
```